addons/point_of_sale/migrations/13.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# Copyright 2020 ForgeFlow <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_unlink_by_xmlid = [
+    # account.journal
+    'point_of_sale.pos_sale_journal',
+    # ir.sequence
+    'point_of_sale.seq_picking_type_posout',
+]
+
+
+def fill_pos_config_default_cashbox_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE pos_config pc
+        SET default_cashbox_id = absc.id
+        FROM account_cashbox_line acl
+        JOIN account_bank_statement_cashbox absc ON acl.cashbox_id = absc.id
+        WHERE acl.{} = pc.id
+        """.format(openupgrade.get_legacy_name('default_pos_id'))
+    )
+
+
+def fill_pos_config_amount_authorized_diff(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE pos_config pc
+        SET amount_authorized_diff = aj.{amount}
+        FROM account_bank_statement abs
+        JOIN account_journal aj ON abs.journal_id = aj.id
+        JOIN pos_session ps ON abs.pos_session_id = ps.id
+        WHERE ps.config_id = pc.id AND pc.amount_authorized_diff IS NULL
+            AND aj.{amount} IS NOT NULL
+        """.format(amount=openupgrade.get_legacy_name('amount_authorized_diff'))
+    )
+
+
+def fill_pos_session_move_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE pos_session ps
+        SET move_id = po.{move}
+        FROM pos_order po
+        WHERE po.session_id = ps.id AND po.{move} IS NOT NULL
+            AND ps.move_id IS NULL AND ps.state = 'closed'
+        """.format(move=openupgrade.get_legacy_name('account_move'))
+    )
```

Comment:
If understood correctly (maybe I'm wrong ...)

- in V12, ``pos_order.account_move`` is a reference to the account_move generated by a pos_order when closing the session.

- in V13 ``pos_session.move_id`` is a reference to the global sale account_move generated when closing the session. 

it doesn't seems to be the same concept. don't you think ?

Reply:
The old `po.invoice_id` is the new `po.account_move`, so the new `ps.move_id` has to be obtained in some way from the old `po.account_move`.